### PR TITLE
Update operator and kubernetes dependencies

### PR DESCRIPTION
**Pull request by howardjohn**
This bumps up the istio/operator dependency, which in turn relies on bumping the whole lot of kubernetes dependencies

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -158,14 +164,60 @@ require (
...
-	k8s.io/apiextensions-apiserver v0.0.0-20190221221350-bfb440be4b87
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
-	k8s.io/cli-runtime v0.0.0-20190221101700-11047e25a94a
-	k8s.io/client-go v10.0.0+incompatible
-	k8s.io/helm v2.13.1+incompatible
+	k8s.io/api v0.0.0
+	k8s.io/apiextensions-apiserver v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/cli-runtime v0.0.0
+	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
```
Comment (hzxuzhonghu) [inline go.mod]: According to https://github.com/kubernetes/client-go, client-go 10.0 is best fit for kubernetes 1.14. Why upgrade client-go singly?

Comment (howardjohn) [reply go.mod]: our kubernetes dependency is 1.15. but we still support older kubernetes versions, just using newer kubernetes library that are backwards compatible.

All of there versions are pinned at kubernetes 1.15

Comment (howardjohn) [reply go.mod]: also this is what is used in operator, so unless we remove istioctl from this repo (which I think we should) we have to match them

File: tools/license/get_dep_licenses.go [hunk 2]
```diff
@@ -123,6 +123,9 @@ var (
 		"github.com/signalfx/com_signalfx_metrics_protobuf": true, // in manual_append
 		"github.com/bmizerany/assert":                       true, // has license in README.md
 		"github.com/kr/logfmt":                              true, // Readme mentions MIT license
+		"bitbucket.org/ww/goautoneg":                        true, // BSD-3-Clause in README
+		"github.com/munnerz/goautoneg":                      true, // BSD-3-Clause in README
+		"github.com/heketi/utils":                           true, // Apache 2.0 in README
```
Comment (geeknoid) [inline tools/license/get_dep_licenses.go]: What are these new imports about?

Comment (howardjohn) [reply tools/license/get_dep_licenses.go]: Kubernetes depends on these for whatever reason, so they are pulled in as transitive dependencies. I went to the repos and they have the licenses mentioned in the README -- not sure if that is the appropriate action here?

**Pull request merged**